### DISCUSSION TITLE
scrolling while generating

### DISCUSSION THREAD

**Discussion by Owen000** [yetone/avante.nvim#826] [Ideas]
is it possible to scroll while generating? seems to lock the cursor essentially

**Comment by tmerse** [comment]
I also don't like this behavior for longer responses.
It is achievable with some small codechanges (see my fork: https://github.com/tmerse/avante.nvim/commit/be75be60fe857962bafa17a4985cdfc1e19e9925)

It would be nice if this could be make configurable via some config flag.

If disabled, I would suggest some spinning indicator to show that an answer is still being generated.

If an answer takes longer or an LLM is slow to respond, I would also suggest to buffer larger chunks and write those bigger chunks at a lower rate, which would make the Avante buffer a bit more interactive (since the cursor is now usable and one may already select text etc.).

**Reply by mblarsen** [reply]
Yeah, it is super annoying. Did you create a PR?

**Comment by mblarsen** [comment]
Was fixed with this https://github.com/yetone/avante.nvim/pull/2227
